### PR TITLE
Overall design rework

### LINKED ISSUE #52: Redesign the page
This is a late issue for @triszt4n 

* The page is too "bootstrapy"
* Primary color is weak, success color is wrong --> rework colorset and overrides
* Use actual logo of SAT instead

### PR DESCRIPTION
**Pull request by triszt4n**
Reason: this project has bootstrap, which is very nice for quick frontend developments, but will get awfully boring UI after a while. This project needed a new image, with better colorsets and moderner (also: cleaner) layouts.

# What's new: (it would be easier to list what's NOT new :sweat_smile: )
* Clean onepager design on frontpage, easier layout for posts.
* Every show, index and edit pages are redesigned equally and in the same manner.
* I forgot, that I'm not writing backend, accidentally implemented an admin-grant endpoints for admins on users index page (The users index page will be useful when doing #71 ).
* Bootstrapped the styling of items pages, just in case (for less work later), but I didn't know how those pages will be used so I used some table for this time (can change later).

I know you guys are not the fans of frontend work, so I will review my own code very carefully, I advise you to review the backend part (and maybe have a look, if the functionalities are working properly as before).

### CODE REVIEW

File: app/views/posts/about.html.erb [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
-        <div class="col-lg-8 col-md-10 text-center">
-          <h1 class="fs-1">Rólunk</h1>
-          <p class="fs-4">Schönherz Airsoft Team</p>
-        </div>
+<div class="row justify-content-center">
+  <div class="col-lg-10 col-12">
+    <div class="d-flex flex-column flex-md-row mt-4 mb-1">
+      <h1 class="text-center text-md-start flex-grow-1">Schönherz Airsoft Team</h1>
+      <div class="text-center ms-md-4">
+        <%= image_tag image_path('favicon.svg'), style: "height: 10rem" %>
```
Comment (bbucsy) [inline app/views/posts/about.html.erb]: We have the logo in the header, and in the footer, and if they want they can put it in the about text. Maybe we don't need it again, and then the `Schönherz Airsoft Team` can be in one line

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -1,8 +1,7 @@
-<nav class="navbar navbar-expand-md bg-secondary navbar-dark" id="mainNav">
+<nav class="navbar navbar-expand-md navbar-light" id="mainNav">
   <div class="container">
     <%= link_to root_path, class: "navbar-brand" do %>
       <%= image_tag image_path('favicon.svg'), size: "32", class: "d-inline-block mr-2" %>
```
Comment (bbucsy) [inline app/views/layouts/_navbar.html.erb]: The design of this logo is... intricate, we could try to make it bigger. Just a suggestion tho

File: app/views/events/show.html.erb [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
+<div class="py-5">
+  <div class="mb-4 mt-2">
+    <%= @event.description %>
   </div>
-  <div class="card-body">
-    <div class="mb-4 mt-2">
-      <%= @event.description %>
+  <% if @event.event_type %>
+    <div class="d-flex justify-content-end mb-2">
+      <%= link_to 'Jelentkezek!', register_to_event_path(@event), class: "btn btn-outline-info" %>
```
Comment (bbucsy) [inline app/views/events/show.html.erb]: This bright blue feels a bit weird, because we don't use the color anywhere else

**Review by bbucsy — APPROVED** [review]
I really like it :+1:
I have written some suggestions, but you can ignore them if you like, I approve this PR

**Review by SepsiLaszlo — APPROVED** [review]
It is stunning!  :star_struck: 
I can't find words to describe how good the new design is! :world_map:

**Pull request merged**